### ISSUE TITLE
Summary of IntelliJ IDEA accessibility issues

### ISSUE THREAD

**Issue by Eskibear** [Microsoft/azure-tools-for-java#1627]
- [x] [IDEA-191322](https://youtrack.jetbrains.com/issue/IDEA-191322) ActionButton & HideableDecorator cannot get focus using tab
Related issues:  ~#1435 #1436 #1438 #1440 #1442 #1508 #1509~

- [x] [IDEA-191311](https://youtrack.jetbrains.com/issue/IDEA-191311) Browse button in widget "com.intellij.openapi.ui.TextFieldWithBrowseButton" cannot get tab focus
Related issues: ~#1440 #1442~
- [x] [IDEA-191315](https://youtrack.jetbrains.com/issue/IDEA-191315) widgets in BaseEditor cannot get focus
Related issues: ~#1444 #1445~

~#1446~

- [ ] [IDEA-191325](https://youtrack.jetbrains.com/issue/IDEA-191325) NVDA doesn't read content inside dialog for version 2017.3 Community
Related issues: #1458 #1459 #1460 #1461 #1462 #1463 #1464 #1465 #1466 #1467 #1468 #1469 #1470 #1471 #1472 #1473 #1474 #1475 #1476 #1477 #1478 #1479 #1483 #1484 #1485 #1486 #1489 #1490 #1493 #1496 #1497 #1500 #1501 #1502 #1503 #1504 #1510 #1511 #1512 #1516 #1518 #1521 #1522 #1523 #1524 

- [x] [IDEA-191682](https://youtrack.jetbrains.com/issue/IDEA-191682) Keyboard focus should be visible on "Help" button
Related issues:  ~#1437 #1439 #1443 #1456~

- [x] [IDEA-191938](https://youtrack.jetbrains.com/issue/IDEA-191938) keyboard focus issue in plugin installation dialogs
Related issues: ~#1453 #1454 #1455~

**Comment by Eskibear** [comment]
As NVDA issues cannot be reproduced in our dev machines. Close it now.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749990
